### PR TITLE
Docs: Update descriptions

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] updates fixer descriptions

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1159,10 +1159,10 @@ Choose from the list of available rules:
 
 * **php_unit_set_up_tear_down_visibility**
 
-  Changes the visibility of the setUp and tearDown functions of phpunit to
-  protected, to match the PHPUnit TestCase.
+  Changes the visibility of the ``setUp()`` and ``tearDown()`` functions of
+  phpunit to ``protected``, to match the PHPUnit TestCase.
 
-  *Risky rule: this fixer may change functions named setUp or tearDown outside of PHPUnit tests, when a class is wrongly seen as a PHPUnit test.*
+  *Risky rule: this fixer may change functions named ``setUp()`` or ``tearDown()`` outside of PHPUnit tests, when a class is wrongly seen as a PHPUnit test.*
```
Comment (localheinz) [inline README.rst]: Ha, not sure about this:

screen shot 2018-03-20 at 22 43 09

Comment (SpacePossum) [reply README.rst]: really like the change, sad that the render is not good :(

File: README.rst [hunk 2]
```diff
@@ -1159,10 +1159,10 @@ Choose from the list of available rules:
 
 * **php_unit_set_up_tear_down_visibility**
 
-  Changes the visibility of the setUp and tearDown functions of phpunit to
-  protected, to match the PHPUnit TestCase.
+  Changes the visibility of the ``setUp()`` and ``tearDown()`` functions of
+  phpunit to ``protected``, to match the PHPUnit TestCase.
```
Comment (julienfalque) [inline README.rst]: _"... functions of **PHPUnit** to..."_

File: README.rst (hunk 1 above)
Comment (julienfalque) [reply README.rst]: Yep, that's sad :/ I think the change here should be reverted.

Comment (SpacePossum) [reply README.rst]: looking at it again, this is already the case for more risky descriptions and I prefer to have it the same way everywhere (and maybe changed everywhere in another PR) than having to forms.

**Comment by SpacePossum** [comment]
note @localheinz I rebased your branch to remove the conflict so be careful when update locally, hope you  don't mind!

**Comment by localheinz** [comment]
@SpacePossum 

Do I need to do anything here? If so, let me please know what it is.

🤓

**Comment by SpacePossum** [comment]
I think we're good here, no @julienfalque ?

**Comment by julienfalque** [comment]
Ah sorry, I thought https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3622#discussion_r178939702 was still to be done.

**Comment by keradus** [comment]
Thank you @localheinz.

**Pull request merged**